projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -27,11 +27,8 @@
   variables:
     _JAVA_OPTIONS: "-Xms2048m -Xmx4096m"
   before_script:
-    - apk --no-cache add git openjdk11-jdk curl --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
-    - curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.21.2/2021-07-05/bin/linux/amd64/aws-iam-authenticator
-    - chmod +x ./aws-iam-authenticator
```

Comment:
> Amazon EKS uses the aws eks get-token command, available in version 1.16.156 or later. Until now we used aws-iam-authenticator but it seems latest changes in Kubernetes do not work there.

Could you link the official docs used in terms of aws-iam-authenticator changes required? For traceability on what has changed, and how

Reply:
They are linked in the PR. All the context I have is in https://github.com/aws/aws-cli/issues/6920 
I am not 100% sure why it happened now. 

I decided to fix it by upgrading the KUBECONFIG and the way its used following  https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html